lib/ansible/modules/cloud/vmware/vmware_datastore_cluster.py

```diff
@@ -34,7 +34,10 @@
     datacenter_name:
       description:
       - The name of the datacenter.
-      required: True
+      - This parameter is optional, if C(folder) is provided.
```

Comment:
If I'm following this correctly, the user must provide either `datacenter_name` or `folder`. If that's correct, then I would suggest we document the requirement this way:

```suggestion
      - You must specify either a C(datacenter_name) or a C(folder).
```

Reply:
OK.